isis/src/base/apps/blend/tsts/default/Makefile

```diff
@@ -7,10 +7,12 @@ I02609002RDR.lev2.blend.cub.TOLERANCE = 0.001
 include $(ISISROOT)/make/isismake.tsts
 
 commands:
-	$(APPNAME) fromlist=$(INPUT)/fromlist.lis \
+	$(LS) $(INPUT)/*.cub > fromlist.lis;
+	$(APPNAME) fromlist=fromlist.lis \
```

Comment:
any particular reason for these type of changes (making the list rather than leaving in the data area)?

Reply:
The files used to be in the $mission/testData dir. Once they were moved to blend/tsts/testName/input the list file has to be generated because of the two different possibilities for the make file $INPUT variable.